src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,50 @@
+.contacts {
+
+  &__block {
+    @include onTablet {
+      display: flex;
+      gap: 75px;
+    }
+  }
+
+  &__data {
+    @include onDesktop {
+      display: flex;
+      gap: 100px;
+    }
+  }
+
+  &__part {
+    margin-top: 48px;
+
+    @include onTablet {
+      display: block;
+      flex-direction: column;
```

Comment:
You don't need flex-direction for block elements.

Reply:
fixed